Garbage collection test fails intermittently if icedtea browser plugin is
installed.

--- a/Source/WebKit/gtk/tests/testdomdocument.c
+++ b/Source/WebKit/gtk/tests/testdomdocument.c
@@ -353,12 +353,6 @@
                test_dom_document_get_links,
                dom_document_fixture_teardown);
 
-    g_test_add("/webkit/domdocument/test_garbage_collection",
-               DomDocumentFixture, HTML_DOCUMENT_LINKS,
-               dom_document_fixture_setup,
-               test_dom_document_garbage_collection,
-               dom_document_fixture_teardown);
-
     return g_test_run();
 }
 
